### PR TITLE
Add styles odt

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

File: template_engines/backends/utils.py [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
 from zipfile import ZipFile
 
 from django.core.files.storage import default_storage
 
 
-def modify_content_document(file_path, xml_path, rendered):
+def get_rendered_by_xml(xml_paths, soup):
+    dict_xml = {}
+    for xml_path in xml_paths:
+        name, _ = os.path.splitext(xml_path)
```
Comment (submarcos) [inline template_engines/backends/utils.py]: attention à _ qui est utilisé pour les traductions e npython. utilier autre chose

**Comment by submarcos** [comment]
add changelog entry

File: template_engines/backends/utils.py (hunk 1 above)
Comment (LePetitTim) [reply template_engines/backends/utils.py]: Done

**Comment by submarcos** [comment]
I'd like that all BeautifulParser are using lxml parser lib, aka xml and lxml, no html.parser directly

**Comment by submarcos** [comment]
https://beautiful-soup-4.readthedocs.io/en/latest/index.html?highlight=lxml#installing-a-parser

**Pull request merged**